packages/wallet/src/json-rpc-validation/validator.ts

```diff
@@ -6,6 +6,8 @@ import getAddressSchema from "@statechannels/client-api-schema/schema/get-addres
 import joinChannelSchema from "@statechannels/client-api-schema/schema/join-channel.json";
 import updateChannelSchema from "@statechannels/client-api-schema/schema/update-channel.json";
 import definitionsSchema from "@statechannels/client-api-schema/schema/definitions.json";
+import channelResultSchema from "@statechannels/client-api-schema/schema/channel-result.json";
+import closeChannelSchema from "@statechannels/client-api-schema/schema/close-channel.json";
```

Comment:
It might be nicer to export the validator from `client-api-schema`, instead of importing all these definitions.

Reply:
I think I agree, I'll do that in a separate PR.